### ISSUE TITLE
Tenir compte du alt_name

### ISSUE THREAD

**Issue by vdct** [osm-fr/bano#27]
Quand le rapprochement échoue sur le name

**Comment by vdct** [comment]
Considérer dans le même ordre d'idée name:left et name:right, cf. https://lists.openstreetmap.org/pipermail/talk-fr/2015-June/076923.html

**Comment by vdct** [comment]
Un cas : 591782240W (Doui - Rue du Jardinage)

**Comment by vdct** [comment]
https://github.com/osm-fr/bano/commit/b5b390b6a94be346cffa384cca0bf4f5069ca198 : alt_name et old_name sont pris en compte sur les noms de voies et noms de points/polygones dans la commune. Les tags ref:FR:FANTOIR, ref:FR:FANTOIR:left, ref:FR:FANTOIR:right ne sont utilisés que pour le tag name.

**Comment by don-vip** [comment]
J'ai des où ça serait bien pratique. Exemple:
315553461Y | 2004-05-27 | RPT FRANCIS CAMBOU | https://www.openstreetmap.org/way/531705282 | name=Rond-Point du Professeur Francis Cambou | alt_name=Rond-Point Francis Cambou

**Comment by vdct** [comment]
Oui, le commit ci-dessus ne faisait qu'une partie du boulot, et la suite n'a jamais été faite. Merci pour le cas de test

**Comment by vdct** [comment]
Tentative de synthèse :
pour une voie donnée, le rapprochement doit pouvoir se faire à l'aide de n'importe lequel de ses noms, au moins parmi name, alt_name, old_name. Le cas des name:right et name:left est différent car il suppose qu'on est à cheval sur 2 communes, le rapprochement doit pouvoir se limiter au nom concernant la commune.
À l'issue du rapprochement, tous les noms sont considérés comme rapprochés. On doit pouvoir donner le type de chacun (name, old, alt), qu'il faut donc conserver jusque dans les données BANO.
Par suite, on doit pouvoir fournir :
- les adresses limitées au rapprochement sur "name" name, lorsqu'on cherche à fournir une liste d'adresses sans doublons (cas du géocodage inverse par exemple)
- toutes les combinaisons de noms et d'adresses, lorsqu'on cherche à maximiser les chances de trouver une adresse (cas du géocodage par exemple)

**Comment by vdct** [comment]
Enfin traité, 10+ ans plus tard....

https://forum.openstreetmap.fr/t/la-gazette-de-pifometre-co-visibilite-des-alt-name-old-name-et-noms-en-langues-regionales/31379
